src/rest_endpoints/file_upload/check.file.exists.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, RequestHandler, Response } from "express";
+//import s3Client from "../../utils/s3.client.js";
```

Comment:
```suggestion

```

Reply:
I commented it out intentionally to prevent the s3 client from initializing because it doesn't have the necessary credentials yet.